custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_battery.yaml

```diff
@@ -28,14 +28,17 @@ card_battery:
       if(variables.ulm_card_battery_charger_type_entity_id == null) {
         // is charging?
         infix = variables.ulm_card_battery_battery_state_entity_id !== null
-                && states[variables.ulm_card_battery_battery_state_entity_id].state =="charging"
+                && states[variables.ulm_card_battery_battery_state_entity_id].state == "charging"
```

Comment:
Could you use `===` for string equality checks? 😄 I think we should update the code base in general to always use `===` where applicable.

Reply:
Bit more background would be nice :-)
https://codeahoy.com/javascript/2019/10/12/==-vs-===-in-javascript/#:~:text=The%20difference%20between%20%3D%3D%20and%20%3D%3D%3D%20is%20that%3A,the%20two%20variables%20being%20compared.

Reply:
We could also covert the state to lowercase and then compare to "charging". This should also remove all problems related to "Charging/charging"

Reply:
Hi @AndyVRD. Do you want to integrate the suggested adjustments?

Reply:
ping @AndyVRD ;-)